ocaml/xapi/xapi_host.ml

```diff
@@ -709,7 +709,19 @@ let evacuate ~__context ~host ~network =
   in
 
   (* avoid edge cases from meaningless batch sizes *)
-  let batch_size = Int.(max 1 (abs !Xapi_globs.evacuation_batch_size)) in
+  let batch_size =
+    Int.(
+      max 1
+        (abs
+           ( match evacuate_batch_size with
+           | Some size ->
+               size
+           | None ->
+               !Xapi_globs.evacuation_batch_size
+           )
+        )
+    )
+  in
```

Comment:
I tend to prefer defining values before using them in expression as they can become unwieldy and more difficult to understand otherwise:
```suggestion
  let batch_size =
    Option.value
      ~default:!Xapi_globs.evacuation_batch_size
      evacuation_batch_size
  in
  (* avoid edge cases from meaningless batch sizes *)
  let batch_size = Int.(max 1 (abs batch_size)) in
```